DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite.reportingengine
 Title: R package to create reports for the Open Systems Pharmacology models
-Version: 2.2.0
+Version: 2.2
```

Comment:
The DESCRIPTION file must remain unchanged. 
The **GitHub-Action** should remove **.0** in the file before calling `build_site()`